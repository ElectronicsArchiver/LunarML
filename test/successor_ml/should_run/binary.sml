print (Int.toString 0b1101 ^ "\n"); (* 0xD = 13 *)
print (Int.toString ~0b1011_1100 ^ "\n"); (* ~0xBC = ~188 *)
print (Word.toString 0wb1110_1111 ^ "\n"); (* 0xEF *)
