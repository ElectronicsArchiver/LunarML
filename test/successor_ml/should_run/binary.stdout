13
~188
EF
